introspection_response_writer_test.go

```diff
@@ -117,8 +120,23 @@ func TestWriteIntrospectionResponseBody(t *testing.T) {
 				sess.SetExpiresAt(ires.TokenUse, time.Time{})
 				ires.AccessRequester = NewAccessRequest(sess)
 			},
-			active: true,
-			hasExp: false,
+			active:   true,
+			hasExp:   false,
+			hasExtra: false,
+		},
+		{
```

Comment:
Could you please add a test which checks that the standard claims are not overwritten by the session? So things like `sub`, `aud`, `expires_at`?